app/controllers/LoginEventController.scala

```diff
@@ -18,14 +18,15 @@ package controllers
 
 import javax.inject.{Inject, Singleton}
 import play.api.mvc.{Action, AnyContent}
-import uk.gov.hmrc.play.frontend.auth.connectors.AuthConnector
+import utils.AuthAction
 import views.html.login_event
+
```

Comment:
any need for whitespace?

Reply:
it was added automatically by IDE, but I will remove

Reply:
done